### PR TITLE
Mention the "show-arguments" option of the "debug:container" command

### LINKED ISSUE #7366: [FrameworkBundle] Document the --show-arguments option of debug:container
See https://github.com/symfony/symfony/pull/20861

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7366.

### CODE REVIEW

**Review by HeahDude — APPROVED** [review]

File: service_container/debug.rst [hunk 1]
```diff
@@ -31,3 +31,13 @@ its id:
 .. code-block:: terminal
 
     $ php bin/console debug:container app.mailer
+
+By default, the arguments of the services are not included in the output. Add the
+``--show-arguments`` option to show them:
+
+.. versionadded:: 3.3
```
Comment (xabbuh) [inline service_container/debug.rst]: Having the `versionadded` directive between the previous paragraph and the output reads a bit weird.

**Comment by wouterj** [comment]
I think the code example can be removed. Maybe we can even do this like:

```rst
.. code-block:: terminal

    $ php bin/console debug:container app.mailer

    # to show the service arguments:
    $ php bin/console debug:container --show-arguments app.mailer

.. versionadded:: 3.3
    The ``--show-arguments`` option was introduced in Symfony 3.3.
```

**Comment by xabbuh** [comment]
Thank you Javier.

**Review by wouterj — APPROVED** [review]

**Comment by xabbuh** [comment]
@wouterj see b395d8dfed14f57adeddf345af0c367a0a37f39e

**Pull request closed without merging**